src/dat/view.cljs

```diff
@@ -266,14 +266,14 @@
 (representation/register-representation
   ::pull-summary-string
   (fn [_ _ pull-data]
-    ;[:span
+    [:span
```

Comment:
Ah; OK. I think (if memory serves me correctly), it's the reference entity selection that needs unwrapped strings in order to be compatible with the `re-com/single-dropdown` `:choices` option. Perhaps the fix is to have `::pull-summary-view` call represent with `::pull-summary-string` as a function instead of as a react component (i.e. `(represent ...)` instead of `[represent ...]`).